components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -312,6 +316,19 @@ abstract class AbstractFetchDownloadService : Service() {
         }
     }
 
+    /**
+     * If notification accent color provided from browser side, returns it. Otherwise returns
+     * default accent color.
+     */
+    data class Style(
+        @ColorRes
+        val notificationAccentColor: Int = R.color.mozac_feature_downloads_notification
+    ) {
+        internal fun getNotificationAccentColor(context: Context): Int {
+            return ContextCompat.getColor(context, notificationAccentColor)
+        }
```

Comment:
As we are just calling `getColor` from `ContextCompat` I think we could remove this function and just call it directly as we were doing in `DownloadNotification`:

`.setColor(ContextCompat.getColor(context, notificationAccentColor))`

Reply:
Done 👍